src/App.js

```diff
@@ -381,15 +416,19 @@ class App extends Component {
                     </div>
                     </div>
                     <div className='header-center'>
-                    <input
+                    <h4>Select Movie by Year</h4><select
                         type="text"
                         ref={this.searchInput}
-                        className="form-controlb"
+                        className="year-selector"
+                        id='mySelect'
                         placeholder="Search by Year 2020-2018"
                         autoFocus={true}
-                        onKeyPress={this.setYear}
-                        // onKeyDown ={this.performList}
-                        />
+                        onChange={this.setYear}
+                        >
+                          <option value = '2020' >2020</option>
+                          <option value = '2019'>2019</option>
+                          <option value = '2018'>2018</option>
+                        </select>
```

Comment:
Move this and place it to the right beside the **Movies Engine Option**, make sure they are both inside the same `div`.

And lastly, hide the **year option filter** on small screen using **CSS media Queries**.

You're amazing man, best of luck.